libs/libcommon/src/libcommon/viewer_utils/features.py

```diff
@@ -76,8 +77,10 @@ def image(
                 image=value,
                 assets_base_url=assets_base_url,
                 assets_directory=assets_directory,
+                overwrite=overwrite,
             )
         except OSError:
+            raise
```

Comment:
why are we changing the logic here? the idea was to try the formats in order.

Reply:
oops it was for debugging